File: tuts/011-getting-started-batch-fargate/java/src/main/java/com/example/batch/GettingStartedScenario.java
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package com.example.batch;

import software.amazon.awssdk.services.batch.BatchClient;

public class GettingStartedScenario {
    public static void main(String[] args) {
        try (BatchClient client = BatchClient.builder().build()) {
            BatchWrapper wrapper = new BatchWrapper(client);
            System.out.println("Running Batch getting started scenario...");
            // TODO: setup, interact, teardown
            System.out.println("Scenario complete.");
        }
    }
}

File: tuts/011-getting-started-batch-fargate/kotlin/src/main/kotlin/com/example/batch/GettingStartedScenario.kt
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package com.example.batch

import aws.sdk.kotlin.services.batch.BatchClient
import kotlinx.coroutines.runBlocking

fun main() = runBlocking {
    BatchClient { region = "us-east-1" }.use { client ->
        val wrapper = BatchWrapper(client)
        println("Running Batch getting started scenario...")
        // TODO: setup, interact, teardown
    }
}
